lib/Basic/LangOptions.cpp

```diff
@@ -111,6 +111,7 @@ static const SupportedConditionalValue SupportedConditionalCompilationPointerBit
 static const SupportedConditionalValue SupportedConditionalCompilationRuntimes[] = {
   "_ObjC",
   "_Native",
+  "_multithreaded",
```

Comment:
If this is the spelling we want, it should probably match the casing of the other valid values.